packages/cli/commands/project/deploy.js

```diff
@@ -42,6 +47,21 @@ exports.handler = async options => {
     projectName = projectConfig.name;
   }
 
+  const projectExists = await ensureProjectExists(accountId, projectName, {
+    allowCreate: false,
+    noLogs: true,
+  });
+
+  if (!projectExists) {
+    logger.error(
+      i18n(`${i18nKey}.errors.projectNotFound`, {
+        projectName: chalk.bold(projectName),
+        accountId: chalk.bold(accountId),
```

Comment:
We have a UI util for referencing an account in the logs: [uiAccountDescription](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli/lib/ui.js#L81)